package.json

```diff
@@ -14,10 +14,14 @@
         "build:android": "npm run build && npx cap sync android && npx cap copy android && npm run open:android",
         "open:android": "npx cap open android",
         "run:android": "npm run build:android && ionic capacitor run android --livereload --external",
-        "release:all:prod": "npm run tag:before:prod && npm run build && firebase use instantvisio && firebase deploy && npm run tag:after",
-        "release:all:dev": "npm run tag:before:dev && firebase use instantvisio-dev &&  env-cmd -f .env.development npm run build && firebase deploy && npm run tag:after",
-        "release:hosting:prod": "npm run tag:before:prod && npm run build && firebase use instantvisio && firebase deploy --only hosting && npm run tag:after",
-        "release:hosting:dev": "npm run tag:before:dev && firebase use instantvisio-dev && env-cmd -f .env.development npm run build && firebase deploy --only hosting",
+        "deploy:all:prod": "npm run build && firebase use instantvisio && firebase deploy",
+        "deploy:all:dev": "firebase use instantvisio-dev &&  env-cmd -f .env.development npm run build && firebase deploy",
+        "release:all:prod": "npm run tag:before:prod && npm run deploy:all:prod && npm run tag:after",
+        "release:all:dev": "npm run tag:before:dev && npm run deploy:all:dev && npm run tag:after",
+        "deploy:hosting:prod": "npm run build && firebase use instantvisio && firebase deploy --only hosting",
+        "deploy:hosting:dev": "firebase use instantvisio-dev &&  env-cmd -f .env.development npm run build && firebase deploy --only hosting",
```

Comment:
shouldn't also `deploy:hosting:prod` contain the `env-cmd -f .env.production` in order to avoid errors ?
In case you `deploy:hosting:dev` first, then try to `deploy:hosting:prod`, the `.env` file would still be the `dev` one if I correctly understood what happens here

Reply:
env-cmd only change fir the current running npm script  so no issue there.